pkg/apis/compliance/v1alpha1/profile_types.go

```diff
@@ -4,6 +4,13 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// PlatformTypeLabel specifies what kind of platform (node,platform)
+// this Profile or a TailoredProfile targets
+const PlatformTypeLabel = "compliance.openshift.io/platform-type"
+// PlatformLabel specifies the name of the platform this Profile
+// or TailoredProfile is targetting. Example: ocp4, rhcos4, ...
+const PlatformLabel = "compliance.openshift.io/platform"
```

Comment:
nit: What if we re-use OpenSCAP terminology here and use `product`? although, platform here is not bad... maybe `platform-id` instead?

Reply:
I like product, because platform is already quite overloaded across CO.